docs/howto/kubernetes/gardener/kubectl.md

```diff
@@ -0,0 +1,95 @@
+# Managing a Kubernetes cluster
+
+## Extracting the kubeconfig file
+
+Once you [have configured a new shoot cluster](create-shoot-cluster), you need to create a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file to be able to access it.
```

Comment:
```suggestion
Once you [have instantiated a new shoot cluster](create-shoot-cluster), you need to create a [kubeconfig](https://kubernetes.io/docs/concepts/configuration/organize-cluster-access-kubeconfig/) file to be able to access it.
```

Since we are configuring *and* creating a new shoot cluster, I suggest a slight change in wording, like the above.

Reply:
I understand why you're not suggesting "created" here (it produces a repetition), but "instantiated" violates Orwell's [Rule № 5](https://en.wikipedia.org/wiki/Politics_and_the_English_Language#Remedy_of_Six_Rules), so I went with "launched".

Reply:
Well, my issue with "configured" was that it's not enough, because you actually configure and **then** create a new shoot cluster. So because that thought stuck in muh mind and wouldn't go away, I came up with "instantiated" and immediately started breathing better :)